### PR TITLE
Feat/flat tab

### PR DESCRIPTION
**Pull request by shivaprasadks**
## Checklist

- [ ] PR is up-to-date with a description of changes and screenshots (if applicable).
- [ ] All files are lint-free.
- [ ] Followed Semantic PR guidelines.
- [ ] Tested locally for regressions.

## Description
This component is another version of the tab component which has flat edges and narrow padding between each tab.
The active tab will be preceded by a blue bar to highlight.
This component can be used for auto tab scroll same as 

## Screenshot
Screenshot 2020-12-02 at 4 12 09 PM

### CODE REVIEW

File: src/components/Tabs/verticalFlatTabs.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.lil-flat-tab-active {
+    background-color: #F8F9FC; 
+    position: relative;
+}
+
+.lil-flat-tab-active:before {
+    top: 0px;  
+    background: #1678FB;
```
Comment (anshul-pinto0410) [inline src/components/Tabs/verticalFlatTabs.css]: kindly use lil-bg class

File: src/components/Tabs/verticalFlatTabs.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+.lil-vertical-flat-tabs > .lil-flat-tab{
+    @apply lil-bg-white lil-opacity-50 lil-py-5 lil-px-0 lil-outline-none lil-flex lil-whitespace-normal lil-cursor-pointer;
+}
+
+.lil-vertical-flat-tabs .lil-flat-tab-active{
+    @apply lil-opacity-100 lil-bg-blue-50;
+}
+
+.lil-flat-tab-active {
+    background-color: #F8F9FC;
```
Comment (anshul-pinto0410) [inline src/components/Tabs/verticalFlatTabs.css]: lil-bg

File: src/components/Tabs/verticalFlatTabs.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+@screen sm {
+    .lil-vertical-flat-tabs {
+        @apply lil-flex lil-overflow-x-auto;
+    }
+
+    .lil-vertical-flat-tabs > .lil-flat-tab{
+        min-width: 300px;
+    }
+}
```
Comment (anshul-pinto0410) [inline src/components/Tabs/verticalFlatTabs.css]: pls add new line

File: src/components/Tabs/verticalFlatTabs.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+.lil-flat-tab-active:before {
+    top: 0px;  
+    background: #1678FB;
+    content: "";
+    @apply lil-absolute lil-h-full lil-inline-block lil-w-1 lil-align-middle;
+}
+
+.lil-tab-lay-height{
+    height: 49rem;
```
Comment (anshul-pinto0410) [inline src/components/Tabs/verticalFlatTabs.css]: where is this being used?

File: src/components/Tabs/verticalFlatTabs.css [hunk 5]
```diff
@@ -11,20 +11,16 @@
 }
 
 .lil-flat-tab-active {
-    background-color: #F8F9FC; 
     position: relative;
```
Comment (anshul-pinto0410) [inline src/components/Tabs/verticalFlatTabs.css:14]: pls use lil-relative

**Review by anshul-pinto0410 — APPROVED** [review]

File: src/components/Tabs/verticalFlatTabs.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    @apply lil-bg-white lil-opacity-50 lil-py-5 lil-px-0 lil-outline-none lil-flex lil-whitespace-normal lil-cursor-pointer;
+}
+
+.lil-vertical-flat-tabs .lil-flat-tab-active{
+    @apply lil-opacity-100 lil-bg-blue-50;
+}
+
+.lil-flat-tab-active {
+    position: relative;
+    @import  lil-bg-grey-190;
```
Comment (pritam248) [inline src/components/Tabs/verticalFlatTabs.css:15]: any reason for using `@import` ? y not `@apply` ?

Comment (shivaprasadks) [reply src/components/Tabs/verticalFlatTabs.css:15]: for the pseudo-selector like :hover, :active, :before to access this class it should be defined as @import , in this case lil-flat-tab-active:before uses this attribute

**Pull request merged**